tests/location.test.js

```diff
@@ -1,42 +1,43 @@
 });
 
 it('Should get all SGW buildings', async () => {
-  const withinBuildingComponent = renderer.create(<WithinBuilding />).getInstance();
+  const withinBuildingComponent = renderer.create(<Location />).getInstance();
   const sgwBuildings = withinBuildingComponent.formatPolygonsObjs('SGW');
   expect(sgwBuildings).toStrictEqual(mockSgwBuildings);
 });
 
 it('Should get all Loyola buildings', async () => {
-  const withinBuildingComponent = renderer.create(<WithinBuilding />).getInstance();
+  const withinBuildingComponent = renderer.create(<Location />).getInstance();
   const loyolaBuildings = withinBuildingComponent.formatPolygonsObjs('LOY');
   expect(loyolaBuildings).toStrictEqual(mockLoyolaBuildings);
 });
 
 it('Should get no buildings with invalid argument', async () => {
-  const withinBuildingComponent = renderer.create(<WithinBuilding />).getInstance();
+  const withinBuildingComponent = renderer.create(<Location />).getInstance();
   const invalidBuildings = withinBuildingComponent.formatPolygonsObjs('UgaBuga');
   expect(invalidBuildings).toStrictEqual([]);
 });
 
 it('Polygon should define points close to edges as inside', async () => {
-  const withinBuildingComponent = renderer.create(<WithinBuilding />).getInstance();
+  const withinBuildingComponent = renderer.create(<Location />).getInstance();
   const bottomLeftPartition = withinBuildingComponent.isInPolygon(4, [0.0, 0.0, 1.0, 1.0], [0.0, 1.0, 0.0, 1.0], 0.1, 0.1);
   const topRightPartition = withinBuildingComponent.isInPolygon(4, [0.0, 0.0, 1.0, 1.0], [0.0, 1.0, 0.0, 1.0], 0.9, 0.9);
-  expect(bottomLeftPartition).toBe(true);
```

Comment:
Keep in mind that the logic of the isInPolygon function is flawed since this test fails even though a square is formed and the point specified for bottomLeftPartition is within the boundaries of the square formed.

Reply:
Ok so it should be reverted right?

Reply:
If not too much of an inconvenience yes.